Rocket.Chat/Controllers/Auth/SignupViewController.swift

```diff
@@ -130,12 +130,14 @@ final class SignupViewController: BaseViewController {
                 }
 
                 AuthManager.auth(email, password: password, completion: { _ in
-                    guard let user = AuthManager.currentUser() else { return }
-                    if user.username != nil {
-                        self?.dismiss(animated: true, completion: nil)
-                        AppManager.reloadApp()
-                    } else {
-                        self?.performSegue(withIdentifier: "RequestUsername", sender: nil)
+                    DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
```

Comment:
Why do we need this call? The Realm call inside of the method is suppose to be happening on the main thread. 🤔